app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -86,3 +86,4 @@ CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
 
 CONFIG_PROBE=y
 CONFIG_PROBE_DMA_MAX=2
+CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL=y
```

Comment:
@abonislawski I'm happy to merge this if this defaults to "n" at the moment until we can resolve all the opens. We also need to add and align all the CPCs on manifest (and that's still WIP, but > 50% done). CI tests for CPC also landing soon, so safe to make default "y" whan all pieces are in place.

Reply:
What's the latest status, do we have any known opens in mainline left? @abonislawski

Reply:
@abonislawski can you confirm this is now working as expected ? I suppose CI will show us if any CPC numbers are worng, but only for tested modules today.

Reply:
It should work correctly, already used in a few MTL releases and we have cpc data for most (or all) of our modules.
Updated today with rebase & review fixes